### PR TITLE
TASK: Remove arguments from VH render methods

### PR DESCRIPTION
**Pull request by kdambekalns**
This removes the arguments from `render()` in Fluid ViewHelpers, and
registers them using `registerArgument` instead.

Arguments to `render()` were deprecated as of Flow 4.0 and support
is removed with Flow 6.0.

### CODE REVIEW

**Comment by kdambekalns** [comment]
Should turn green once the Flow change is merged (https://github.com/neos/flow-development-collection/pull/1578), I think. With all changes applied I have it working locally, at least.

File: Neos.Media.Browser/Classes/ViewHelpers/PaginateViewHelper.php [hunk 1]
```diff
@@ -37,19 +36,29 @@ class PaginateViewHelper extends AbstractWidgetViewHelper
...
 
+    /**
+     * Initialize the arguments.
+     *
+     * @return void
+     * @throws \Neos\FluidAdaptor\Core\ViewHelper\Exception
+     */
+    public function initializeArguments()
+    {
+        $this->registerArgument('queryString', 'string', '', true);
```
Comment (daniellienert) [inline Neos.Media.Browser/Classes/ViewHelpers/PaginateViewHelper.php]:
```suggestion
        $this->registerArgument('queryResult', AssetProxyQueryResultInterface::class, '', true);
```

File: Neos.Neos/Classes/ViewHelpers/Link/ModuleViewHelper.php [hunk 2]
```diff
@@ -51,27 +51,28 @@ public function initializeArguments()
...
      * @return string The rendered link
      * @throws \Neos\FluidAdaptor\Core\ViewHelper\Exception
      */
-    public function render($path, $action = null, $arguments = [], $section = '', $format = '', array $additionalParams = [], $addQueryString = false, array $argumentsToBeExcludedFromQueryString = [])
+    public function render(): string
     {
         $this->uriModuleViewHelper->setRenderingContext($this->renderingContext);
 
-        $uri = $this->uriModuleViewHelper->render($path, $action, $arguments, $section, $format, $additionalParams, $addQueryString, $argumentsToBeExcludedFromQueryString);
+        $uri = $this->uriModuleViewHelper->render($this->arguments['path'], $this->arguments['action'], $this->arguments['arguments'], $this->arguments['section'], $this->arguments['format'], $this->arguments['additionalParams'], $this->arguments['addQueryString'], $this->arguments['argumentsToBeExcludedFromQueryString']);
```
Comment (daniellienert) [inline Neos.Neos/Classes/ViewHelpers/Link/ModuleViewHelper.php]: I fear that's not working. You have to hand over the parameters via `setArguments()`

Comment (kdambekalns) [reply Neos.Neos/Classes/ViewHelpers/Link/ModuleViewHelper.php]: Ouch. Obviously. 💀

**Comment by kdambekalns** [comment]
> Looks good, Neos backend works when both changes are applied.

@daniellienert – could you approve the Flow PR as well, please?

**Comment by daniellienert** [comment]
@kdambekalns: Reviewed and merged. Lets see if the tests turn green.

**Pull request merged**